### DISCUSSION TITLE
Crazyswarm Error issue

### DISCUSSION THREAD

**Discussion by WoodyloveBo** [bitcraze/discussions#1089] [Q&A]
hello. I'm trying to control a Crazyflie by following Crazyswarm's tutorial.
Currently, I have confirmed that crazyflie is displayed in rviz, including rolaunch crazyswarm hover_swarm.launch.
But when I enter the python3 file, the run doesn't run like this: What am I to the heart of the matter?
![KakaoTalk_20231214_151723407](https://github.com/bitcraze/discussions/assets/143582301/0a461514-2a3f-4659-839a-94c360a856bd)

**Comment by knmcguire** [comment]
Hi! Any support questions for crazyswarm needs to go to their discussions: https://github.com/USC-ACTLab/crazyswarm/discussions

However, if you are not fixed on the project, I would very much advise you to go switch to crazyswarm 2. There is a lot more support there now than crazyswarm1: https://imrclab.github.io/crazyswarm2/

This is crazyswarm2's discussion forum: https://github.com/IMRCLab/crazyswarm2/discussions
